src/test/java/com/animalfinder/enterpriseproject/BreedServiceTest.java

```diff
@@ -25,22 +25,22 @@
 	IBreedService breedService;
 	List<AnimalDTO> animal;
 	
-	
+	//added an "o" to fetchBreed_ValidateNoResultsForJunkData()

```

Comment:
Good changes... but again, don't put discussion/rationale in source code comments.  In the pull request, you can have a conversation with multiple parties, without cluttering up code.